Firestore/Example/SwiftBuildTest/main.swift

```diff
@@ -228,11 +228,11 @@ func readDocument(at docRef: DocumentReference) {
 func readDocumentWithOptions(at docRef: DocumentReference) {
   docRef.getDocument(options:GetOptions.defaultOptions()) { document, error in
   }
-  docRef.getDocument(options:GetOptions.init(source:Source.default)) { document, error in
+  docRef.getDocument(options:GetOptions.init(source:GetSource.default)) { document, error in
```

Comment:
Try making one of these use the unqualified form:

```
docRef.getDocument(options:GetOptions(source:.default)) { // ...
```

Also, explicitly calling init is not required. Using a constructor-style invocation calls the init function.

Reply:
Done.